src/js/keyboardInput.js

```diff
@@ -128,7 +135,39 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             "onCreate.registerChangeListener": {
                 funcName: "gpii.firstDiscovery.keyboardInput.registerChangeListener",
                 args: ["{that}", "{that}.container"]
+            },
+            // begin TOOLTIP HANDLER CONFIGURATION
+            //
+            // We want to control the tooltip opening ourselves so we
+            // remove the existing mouseover and focusin handlers and
+            // add our own instead.  We leave the mouseleave and
+            // focusout handlers alone as the jQuery Tooltip widget
+            // rebinds these each time the tooltip is opened.
+            "onCreate.removeMouseover": {
+                "this": "{that}.container",
+                method: "off",
+                args: ["mouseover"],
+                priority: 4
+            },
+            "onCreate.removeFocusin": {
+                "this": "{that}.container",
+                method: "off",
+                args: ["focusin"],
+                priority: 3
+            },
+            "onCreate.mouseoveHandler": {
```

Comment:
Fixed priorities are ok here, but I think we only need 2 of them in order to avoid "excess intention". We should leave a note here to replace this with a FLUID-5506 constraint-based priority once that is merged in. We should also take care to add namespaces (that is, jQuery event namespaces) to the raw mouseover and focusin handlers so that we can identify them precisely later if necessary. There's a typo in the Infusion namespace here, "mouseoveHandler".